src/item.h

```diff
@@ -2293,7 +2293,7 @@ class item : public visitable
         const std::vector<comp_selection<tool_comp>> &get_cached_tool_selections() const;
 
         std::vector<enchantment> get_enchantments() const;
-        double calculate_by_enchantment( const Character &owner, double modify, enchant_vals::mod value,
+        double calculate_by_enchantment( Character &owner, double modify, enchant_vals::mod value,
```

Comment:
calculations like this should not mutate the character. we do not want to remove the const here lightly.